images/Document about me_files/styles.css

```diff
@@ -0,0 +1,151 @@
+body {
```

Comment:
This file and all of the files in this folder seem to not be used in the final webpage. It would be good to remove all of these before submitting projects so that the code is clean and it is clear what code is part of the project.